pageserver/src/tenant/timeline/compaction.rs

```diff
@@ -735,10 +745,12 @@ impl Timeline {
                 if !self.cancel.is_cancelled() && !err.is_cancelled() {
                     tracing::error!("could not compact, repartitioning keyspace failed: {err:?}");
                 }
-                (1, false)
+                1
             }
         };
 
+        // 3. Shard ancestor compaction
```

Comment:
nit: there is another 3 above, maybe remove it since it's now covered by 2?

Reply:
This should be (4) indeed